src/Microsoft.VisualBasic/src/Microsoft/VisualBasic/FileIO/FileSystem.Portable.vb

```diff
@@ -0,0 +1,1091 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Option Strict On
+Option Explicit On
+
+Imports Microsoft.VisualBasic.CompilerServices
+Imports Microsoft.VisualBasic.CompilerServices.Utils
+Imports System.Collections.Specialized
+Imports System.ComponentModel
+Imports System.Runtime.InteropServices
+Imports System.Security
+Imports System.Text
+Imports ExUtils = Microsoft.VisualBasic.CompilerServices.ExceptionUtils
+
+'''' IMPORTANT: Changes made to public interface of FileSystem should be reflected in Ref/Microsoft.VisualBasic.cs
+Namespace Microsoft.VisualBasic.FileIO
+
+    ''' <summary>
+    '''  This class represents the file system on a computer. It allows browsing the existing drives, special directories;
+    '''  and also contains some commonly use methods for IO tasks.
+    ''' </summary>
+    Partial Public Class FileSystem
```

Comment:
>FileSystem [](start = 25, length = 10)

It looks like this file was created by splitting `FileSystem.vb`. Is there a reason to split the file? If not, please consider merging back so it's easier to compare the ported file with the reference source.

Reply:
The original intent was to put OS specific code before I know about Partial Classes now it serves no purpose and I will merge back next week.